scripts/pylint_extensions_test.py

```diff
@@ -1046,3 +1046,40 @@ def test_no_space_after_yield_statement(self):
             ),
         ):
             temp_file.close()
+
+
+class ExcessiveEmptyLinesCheckerTests(unittest.TestCase):
```

Comment:
Add more tests cases:
- Check that if there are only 2 lines, it doesn't error
- Check that if there are more places that violate this error, multiple errors are spit out.

Reply:
Done.